yoti-sdk-sandbox/pom.xml

```diff
@@ -0,0 +1,314 @@
+  <name>Yoti SDK sandbox package</name>
+  <description>Java SDK for simple integration with the Sandbox Yoti platform</description>
+  <url>https://github.com/getyoti/yoti-java-sdk</url>
+
+  <developers>
+  </developers>
+
+  <properties>
+    <yoti.sdk.version>1.4.3-SNAPSHOT</yoti.sdk.version>
+  
+    <guava.version>20.0</guava.version>
+    <maven.compiler.source>1.6</maven.compiler.source>
+    <maven.compiler.target>1.6</maven.compiler.target>
+    <owasp.dependency.check.cvss.limit>4</owasp.dependency.check.cvss.limit>
+    <owasp.dependency.check.cve.update.limit>24</owasp.dependency.check.cve.update.limit>
+    <findbugs.version>3.0.0</findbugs.version>
+  </properties>
+
+  <licenses>
+    <license>
+      <name>Yoti License</name>
+      <url>https://github.com/getyoti/yoti-java-sdk/LICENSE.txt</url>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:http://github.com/getyoti/yoti-java-sdk.git</connection>
+    <developerConnection>scm:http://github.com/getyoti/yoti-java-sdk.git</developerConnection>
+    <url>http://github.com/getyoti/yoti-java-sdk.git</url>
+  </scm>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.hamcrest</groupId>
+        <artifactId>hamcrest-core</artifactId>
+        <version>1.3</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>1.59</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.7.9.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>3.5.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.11</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.25</version>
+    </dependency>
+    <dependency>
+      <groupId>com.yoti</groupId>
+      <artifactId>yoti-sdk-api</artifactId>
+      <version>${yoti.sdk.version}</version>
+    </dependency>
+    <!-- Testing dependencies -->
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-library</artifactId>
+      <version>1.3</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>2.8.47</version>
```

Comment:
is not this version a bit old? At least for test dependencies we'd keep the versions as much updated as possible. The new version of mockito gives important info, whilst the tests run, about unused Mock obj